test/iteration.jl

```diff
@@ -9,16 +9,20 @@ module TestIteration
 
     @test size(eachrow(df)) == (size(df, 1),)
     @test eachrow(df)[1] == DataFrameRow(df, 1)
+    @test collect(eachrow(df)) isa Vector{DataFrameRow{DataFrame}}
+    @test eltype(collect(eachrow(df))) == DataFrameRow{DataFrame}
```

Comment:
This test is redundant with the previous one, I meant calling `eltype(eachrow(df))`.

Reply:
OK - now I get it. Will fix it.